concepts/structs/about.md

```diff
@@ -1,34 +1,36 @@
 # About
+
 In Go, a `struct` is a sequence of named elements called _fields_, each field has a name and type. The name of a field must be unique within the struct.
 Structs can be compared with _classes_ in the Object-Oriented Programming paradigm.
 
 You create a new struct by using the `type` keyword and the **_built-in type_** `struct`, and explicitly define the name and type of the fields.
-For example, to define a `Person` struct:
+For example, to define a `Shape` struct:
```

Comment:
I changed the struct to `Shape`, because later in the constructor, I use a default-value:
```
func NewShape(name string) Shape {
	return Shape{
		name: name,
		size: 100, //default-value for size is 100
	}
}
```

It was hard to find a default value for `Person` that is not somehow political or racist or something like this.